src/tests/utils/ApiTestSteps.ts

```diff
@@ -508,3 +509,17 @@ export function validateCriVcIssuedTxMAEvent(txmaEvent: any, yotiMockId: any): a
 			console.warn("Yoti Mock Id provided does not match expected list");
 	}
 } 
+
+//F2F-948
```

Comment:
I think a comment for @shenry99 do we need this code comment? The git blame should show anyone that needs to know what ticket all of these changes are linked to (especially if we include ticket numbers in our commit messages)